lib/get/get-full-source-space.js

```diff
@@ -72,7 +73,7 @@ export default function getFullSourceSpace ({
       title: 'Fetching content entries data',
       task: (ctx, task) => {
         const teardownTaskListeners = logToTaskOutput(task)
-        return pagedGet(ctx.space, 'getEntries')
+        return pagedGet(ctx.space, 'getEntries', 0, null, query)
```

Comment:
Having to pass *five* args in the right order is a sign for me to refactor it to an `options` arg, so that you have "named" arguments, WDYT?
(I would sacrifice myself to change it in the codebase, only found 7 occurences)

Reply:
You are right, I can do it